Rock Paper Scissors/rock-paper-scissors.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock Paper Scissors Game</title>
+</head>
+<body>
+    <script>
+        let computerScore = 0;
+        let userScore = 0;
+
+        function computerPlay() {
+            const choices = ["Rock", "Paper", "Scissors"];
+            const randomNumber = Math.floor(Math.random() * 3);
+            return choices[randomNumber];
+        }
+
+        function playRound() {
```

Comment:
What if the user presses Cancel at the prompt?

Reply:
struggling on this one at the moment.. I ve managed to handle the case where the user clicks on cancel but now I am facing a new problem, when I give a correct value in order to make it work I need to reassign the variable userInput again with a prompt otherwise it goes on an infinite loop.  I haven't managed to pass the value of userInput from the function game which runs first to the playround function, I think there is the solution but I haven't to solve this yet, I will put it as a question to the weekly meeting tomorrow and with the help of my teammates I will try to solve it otherwise I ll send you the code to see where I am at the moment.. have a good night or a good day.